content/general/concepts/number/number.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: 'Number'
+Description: 'The number data type encompasses many numeric values.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. When they are computed, there is a trade-off in accuracy for range. Which means they trade-off how acute it is with how many numbers it can compute.
```

Comment:
I think you're trying to say there is a trade-off in precision for range. Precision is how many bits are in the floating-point number representation. The range is the number of values that can be represented. The smaller the precision, the smaller the range in values we can have. And vice versa, the bigger the precision, the bigger the range in values. Precision does have an effect on accuracy (how close to the real value you can get) though.
```suggestion
Numbers include `-1`, `0`, `1`, and so on. Depending on the language, numbers with decimals are of a different type. In floating-point arithmetic, there is a trade-off in precision for range. The amount of bits allocated for floating-point representation affects the range in values that can be used.
```